lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/full_page_embedding_strategy.dart

```diff
@@ -48,6 +48,12 @@ class FullPageEmbeddingStrategy extends EmbeddingStrategy {
     registerElementForCleanup(resourceHost);
   }
 
+  @override
+  void disableContextMenu() => disableContextMenuOn(domDocument.body!);
```

Comment:
We usually set these event listeners on `window` not `document.body`. In order to do that, you want to change the signature of `disableContextMenuOn`/`enableContextMenuOn` to take a `DomEventTarget` instead of a `DomElement`. Then do:
```suggestion
  void disableContextMenu() => disableContextMenuOn(domWindow);
```